submodules/irsa/variables.tf

```diff
@@ -0,0 +1,85 @@
+variable "deploy_id" {
+  type        = string
+  description = "Domino Deployment ID"
+
+  validation {
+    condition     = can(regex("^[a-z-0-9]{3,32}$", var.deploy_id))
+    error_message = "Argument deploy_id must: start with a letter, contain lowercase alphanumeric characters(can contain hyphens[-]) with length between 3 and 32 characters."
+  }
+}
+
+variable "storage_info" {
+  description = "Storage info."
```

Comment:
This is pretty generic, we should add a little more declarative information to clarify its purpose.